### DISCUSSION TITLE
Problems plotting orientations into inverse pole figure

### DISCUSSION THREAD

**Discussion by Kequ90** [mtex-toolbox/mtex#1949] [Ask Anything]
Hey,

I got some orientation data in matrix form and I want to plot them into an inverse pole figure, just like its shown here (https://mtex-toolbox.github.io/EBSDOrientationPlots.html) in the "scatter (inverse) pole figure plot"...

So what i do is just:
`mydata=someorientationdata` as 3x3xN matrix
```matlab
ori=orientation.byMatrix(mydata);
cs = crystalSymmetry('432');

r = [vector3d.X,vector3d.Y,vector3d.Z];
plotIPDF(ori,r)
```

According to all the documentations this houkld result in 3 inverse pole figure plot, showing the Poles of the N orientations.

Instead i juts get the whole polefigure (not reduced to the standard triangle) as if I would use plotPDF... So what am I doing wrong?

I'm using MTEX version 5.10.0

Many thanks in advance and best regards

**Comment by ralfHielscher** [comment]
You need to specify the crystal symmetry when defining the orientation, i.e.,
```matlab
cs = crystalSymmetry('432');
ori=orientation.byMatrix(mydata,cs);

r = [vector3d.X,vector3d.Y,vector3d.Z];
plotIPDF(ori,r)
```
Ralf.
